backend/CONTRIBUTING.md

```diff
@@ -1,3 +1,39 @@
 # Contributing
 
-TODO
+## Install and run locally
+
+```
+docker-compose run backend
+```
+The API then becomes available at [http://localhost:8081]()
+
+## Stack & Libraries
+The backend of the app uses Flask, uWSGI and Nginx to serve the API.
+Prefer open-source libraries when possible.
+
+## Backend endpoints
+The backend API has the following endpoints and methods:
+
+| Endpoint                  |      Method  |  Options |
+|-------------------------- |:------------:|:------|
+| /available-service-areas|  GET         |  |
+| /api/providers     |  POST            | body: providers=[{"provider_address"}] |
+| /api/representative_points     |  POST            | body: representative_points=[{rep_point_ids}] |
+| /api/adequacies     |  POST            | body: {providers=[{provider_ids}], representative_points=[{rep_point_ids}]} |
+
+## Docstring
+When contributing, please be mindful of people coming after you and write down clear docstrings. We suggest the following schema for docstrings:
```

Comment:
Can we enforce this with a linter instead?

Reply:
Will create an issue for finding good additional linters.